deluge/core/core.py

```diff
@@ -359,10 +362,12 @@ def do_remove_torrents():
             for torrent_id in torrent_ids:
                 try:
                     self.torrentmanager.remove(torrent_id, remove_data=remove_data, save_state=False)
-                except InvalidTorrentError:
-                    errors.append(torrent_id)
+                except InvalidTorrentError as ex:
+                    errors.append((torrent_id, str(ex)))
             # Save the session state
             self.torrentmanager.save_state()
+            if errors:
+                log.error("Failed to remove %d of %d torrents.", len(errors), len(torrent_ids))
```

Comment:
Well, the main reason is that it shouldn't really be happening, so at least it informs us that something isn't right